content/en/data_jobs/airflow.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Enable Data Jobs Monitoring for Apache Airflow
+is_beta: true
+private: true
+aliases:
+    - /data_jobs/airflow/
+    - data_jobs/airflow/
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="#" header="false" btn_hidden="true" >}}
+  Data Jobs Monitoring for Apache Airflow is in preview. To sign up for access, fill out [this form](https://www.datadoghq.com/private-beta/monitoring-for-data-and-data-pipelines/).
```

Comment:
```suggestion
  Data Jobs Monitoring for Apache Airflow is in Preview. To sign up for access, fill out [this form](https://www.datadoghq.com/private-beta/monitoring-for-data-and-data-pipelines/).
```

Reply:
Hi @rtrieu , can you update the link to `https://www.datadoghq.com/private-beta/monitoring-for-data-and-data-pipelines/` instead?

Reply:
Oops, yes, will update my suggestion.